sdk/textanalytics/Azure.AI.TextAnalytics/src/DocumentSentimentLabel.cs

```diff
@@ -4,9 +4,9 @@
 namespace Azure.AI.TextAnalytics
 {
     /// <summary>
-    /// The predicted sentiment label for a given span of text.
+    /// The predicted sentiment label for a given document.
     /// </summary>
-    public enum SentimentLabel
+    public enum DocumentSentimentLabel
     #pragma warning restore
```

Comment:
I may be missing something, but I don't see a matching warning suppression.   Is this still needed?